src/libraries/System.Reflection.Metadata/README.md

```diff
@@ -0,0 +1,10 @@
+# System.Reflection.Metadata
+This provides a low-level API for inspecting and writing raw assembly contents according to ECMA-335. Assemblies and types are not loaded into the current runtime context.
```

Comment:
Compare this document with the official documentation. The official documentation just says "Contains types that represent metadata information about assemblies, modules, members, parameters, and other entities in managed code." and gives you very long list of types. 

This documentation looks much better, e.g. it tells about the most important types.

I would argue we should be improving official documentation, and this readme should just say "Documentation at https://learn.microsoft.com/dotnet/api/system.reflection.metadata"

Reply:
(Similar for other readmes.)

Reply:
That sounds good to me. It looks like it's just an update to these:
https://github.com/dotnet/dotnet-api-docs/blob/main/xml/ns-System.Reflection.Metadata.xml

Reply:
Sure - I removed that and will create another PR shortly for that doc. Thanks

Reply:
See https://github.com/dotnet/dotnet-api-docs/pull/8476